lib/providers/socket/commons.js

```diff
@@ -24,6 +24,19 @@ exports.setupMethodHandler = function setupMethodHandler (emitter, params, servi
         args.splice(position, 0, {});
       }
       args[position] = _.extend({ query: args[position] }, params);
+
+      // If there was no callback supplied by a socket client...
```

Comment:
Can we put this into a separate PR since it belongs to some different functionality?